File: loop_transformations/sources/apply_syntax.3.c
/*
* @@name:	apply_syntax.3
* @@type:	C
* @@operation:	compile
* @@expect:	success
* @@version:	omp_6.0
*/
void apply_complexarg(double A[100*100])
{
   #pragma omp tile sizes(4,5) \
      apply(grid(1): reverse)  \
      apply(intratile(2): unroll)
   for (int i = 0; i < 100; ++i)
      for (int j = 0; j < 100; ++j)
         A[i*100+j] += 1;
}

File: loop_transformations/sources/apply_syntax.3.f90
! @@name:	apply_syntax.3
! @@type:	F-free
! @@operation:	compile
! @@expect:	success
! @@version:	omp_6.0
subroutine apply_complexarg(A)
   implicit none
   double precision :: A(100,100)
   integer :: i, j

   !$omp tile sizes(4,5)            &
   !$omp&   apply(grid(1): reverse) &
   !$omp&   apply(intratile(2): unroll)
   do i = 1, 100
      do j = 1, 100
         A(j,i) = A(j,i) + 1
      end do
   end do
end subroutine
